playbooks/scripts/required_user_config_keys.py

```diff
@@ -31,6 +31,11 @@
 be considered an exception.
 """
 
+# This addresses RLM 1374, proper indentation for mapped sequence blocks
+class IndentFix(yaml.Dumper):
```

Comment:
There is supposed to be 2 blank line before class declaration, which is required by pep8.

Reply:
Also class comment can be addressed below class declaration line.

class WithADocString(object):

    """Summary line.

    Bla bla bla bla.
    """